models using cvodes compilation broken on Linux

#### Summary:
Compilation of [edit:some] models using cvodes is broken in 2.20.0 on Linux. SIR seems fine but one_comp_mm_elim_abs doesn't compile. 

#### Description:
```
g++ -march=core2 -std=c++1y -pthread -Wno-sign-compare     -O3 -I src -I stan/src -I stan/lib/stan_math/ -I stan/lib/stan_math/lib/eigen_3.3.3 -I stan/lib/stan_math/lib/boost_1.69.0 -I stan/lib/stan_math/lib/sundials_4.1.0/include    -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DBOOST_PHOENIX_NO_VARIADIC_EXPRESSION                   src/cmdstan/main.o stan/lib/stan_math/lib/sundials_4.1.0/lib/libsundials_nvecserial.a stan/lib/stan_math/lib/sundials_4.1.0/lib/libsundials_cvodes.a stan/lib/stan_math/lib/sundials_4.1.0/lib/libsundials_idas.a  ../stat_comp_benchmarks/benchmarks/pkpd/one_comp_mm_elim_abs.o -o ../stat_comp_benchmarks/benchmarks/pkpd/one_comp_mm_elim_abs
../stat_comp_benchmarks/benchmarks/pkpd/one_comp_mm_elim_abs.o: In function `std::vector<std::vector<stan::return_type<double, double, double, double>::type, std::allocator<stan::return_type<double, double, double, double>::type> >, std::allocator<std::vector<stan::return_type<double, double, double, double>::type, std::allocator<stan::return_type<double, double, double, double>::type> > > > stan::math::cvodes_integrator<2>::integrate<one_comp_mm_elim_abs_model_namespace::one_comp_mm_elim_abs_functor__, double, double, double, double>(one_comp_mm_elim_abs_model_namespace::one_comp_mm_elim_abs_functor__ const&, std::vector<double, std::allocator<double> > const&, double const&, std::vector<double, std::allocator<double> > const&, std::vector<double, std::allocator<double> > const&, std::vector<double, std::allocator<double> > const&, std::vector<int, std::allocator<int> > const&, std::ostream*, double, double, long)':
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x495): undefined reference to `N_VMake_Serial'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x4b9): undefined reference to `SUNDenseMatrix'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x4d1): undefined reference to `SUNDenseLinearSolver'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x4f4): undefined reference to `CVodeCreate'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x5be): undefined reference to `CVodeInit'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x5d9): undefined reference to `CVodeSetUserData'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x60f): undefined reference to `CVodeSetErrHandlerFn'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x621): undefined reference to `CVodeSStolerances'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x63a): undefined reference to `CVodeSetMaxNumSteps'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x654): undefined reference to `CVodeSetInitStep'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x66f): undefined reference to `CVodeSetMaxErrTestFails'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x68a): undefined reference to `CVodeSetMaxConvFails'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x6b2): undefined reference to `CVodeSetLinearSolver'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x6d1): undefined reference to `CVodeSetJacFn'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x74e): undefined reference to `CVode'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x866): undefined reference to `CVodeFree'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x873): undefined reference to `SUNLinSolFree'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x880): undefined reference to `SUNMatDestroy'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x88d): undefined reference to `N_VDestroy_Serial'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x931): undefined reference to `N_VCloneVectorArrayEmpty_Serial'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0x999): undefined reference to `N_VDestroyVectorArray_Serial'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0xb42): undefined reference to `CVodeFree'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0xc41): undefined reference to `SUNLinSolFree'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0xc4e): undefined reference to `SUNMatDestroy'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0xc5b): undefined reference to `N_VDestroy_Serial'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EddddEESt6vectorIS6_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISD_EESaISF_EERKT_RKS6_IS8_SaIS8_EERKSA_RKS6_ISB_SaISB_EERKS6_IS9_SaIS9_EERKS6_IdSaIdEERKS6_IiSaIiEEPSoddl]+0xc75): undefined reference to `N_VDestroyVectorArray_Serial'
../stat_comp_benchmarks/benchmarks/pkpd/one_comp_mm_elim_abs.o: In function `std::vector<std::vector<stan::return_type<double, stan::math::var, double, double>::type, std::allocator<stan::return_type<double, stan::math::var, double, double>::type> >, std::allocator<std::vector<stan::return_type<double, stan::math::var, double, double>::type, std::allocator<stan::return_type<double, stan::math::var, double, double>::type> > > > stan::math::cvodes_integrator<2>::integrate<one_comp_mm_elim_abs_model_namespace::one_comp_mm_elim_abs_functor__, double, stan::math::var, double, double>(one_comp_mm_elim_abs_model_namespace::one_comp_mm_elim_abs_functor__ const&, std::vector<double, std::allocator<double> > const&, double const&, std::vector<double, std::allocator<double> > const&, std::vector<stan::math::var, std::allocator<stan::math::var> > const&, std::vector<double, std::allocator<double> > const&, std::vector<int, std::allocator<int> > const&, std::ostream*, double, double, long)':
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x61a): undefined reference to `N_VMake_Serial'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x63e): undefined reference to `SUNDenseMatrix'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x656): undefined reference to `SUNDenseLinearSolver'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x679): undefined reference to `CVodeCreate'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x74e): undefined reference to `CVodeInit'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x771): undefined reference to `CVodeSetUserData'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x7b0): undefined reference to `CVodeSetErrHandlerFn'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x7c2): undefined reference to `CVodeSStolerances'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x7db): undefined reference to `CVodeSetMaxNumSteps'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x7f5): undefined reference to `CVodeSetInitStep'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x810): undefined reference to `CVodeSetMaxErrTestFails'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x82b): undefined reference to `CVodeSetMaxConvFails'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x856): undefined reference to `CVodeSetLinearSolver'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x878): undefined reference to `CVodeSetJacFn'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x91c): undefined reference to `CVodeGetSens'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0xf44): undefined reference to `CVode'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0xf64): undefined reference to `CVodeFree'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0xf71): undefined reference to `SUNLinSolFree'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0xf7e): undefined reference to `SUNMatDestroy'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0xf8b): undefined reference to `N_VDestroy_Serial'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x105c): undefined reference to `N_VCloneVectorArrayEmpty_Serial'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x10da): undefined reference to `CVodeSensInit'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x10f5): undefined reference to `CVodeSensEEtolerances'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x1115): undefined reference to `N_VDestroyVectorArray_Serial'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x1218): undefined reference to `SUNLinSolFree'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x1225): undefined reference to `SUNMatDestroy'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x1232): undefined reference to `N_VDestroy_Serial'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x124c): undefined reference to `N_VDestroyVectorArray_Serial'
one_comp_mm_elim_abs.hpp:(.text._ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl[_ZN4stan4math17cvodes_integratorILi2EE9integrateIN36one_comp_mm_elim_abs_model_namespace30one_comp_mm_elim_abs_functor__EdNS0_3varEddEESt6vectorIS7_INS_11return_typeIT0_JT1_T2_T3_EE4typeESaISE_EESaISG_EERKT_RKS7_IS9_SaIS9_EERKSB_RKS7_ISC_SaISC_EERKS7_ISA_SaISA_EERKS7_IdSaIdEERKS7_IiSaIiEEPSoddl]+0x12ec): undefined reference to `CVodeFree'
collect2: error: ld returned 1 exit status
make/program:38: recipe for target '../stat_comp_benchmarks/benchmarks/pkpd/one_comp_mm_elim_abs' failed

```

#### Reproducible Steps:
Try compiling the one_comp_mm_elim_abs model from stat_comp_benchmarks on Linux.

#### Current Version:
v2.20.0

--- a/make/program
+++ b/make/program
@@ -38,7 +38,7 @@ endif
 	@echo ''
 	@echo '--- Compiling, linking C++ code ---'
 	$(COMPILE.cpp) $(CXXFLAGS_PROGRAM) -x c++ -o $*.o $<
-	$(LINK.cpp) $(LDLIBS) $(CMDSTAN_MAIN_O) $(LIBSUNDIALS) $(MPI_TARGETS) $*.o $(OUTPUT_OPTION)
+	$(LINK.cpp) $*.o $(CMDSTAN_MAIN_O) $(LDLIBS) $(LIBSUNDIALS) $(MPI_TARGETS) $(OUTPUT_OPTION)
 
 
 ##

